lib/hls/hls_parser.js

```diff
@@ -1934,6 +1900,64 @@ shaka.hls.HlsParser = class {
     };
   }
 
+  /**
+   * @param {!shaka.hls.Playlist} playlist
+   * @param {string} mimeType
+   * @private
+   */
+  parseDrmInfo_(playlist, mimeType) {
+    /** @type {!Array.<!shaka.hls.Tag>} */
+    const drmTags = [];
+    if (playlist.segments) {
+      for (const segment of playlist.segments) {
+        const segmentKeyTags = shaka.hls.Utils.filterTagsByName(segment.tags,
+            'EXT-X-KEY');
+        drmTags.push(...segmentKeyTags);
+      }
+    }
+
+    let encrypted = false;
+    let aesEncrypted = false;
+
+    /** @type {!Array.<shaka.extern.DrmInfo>}*/
+    const drmInfos = [];
+    const keyIds = new Set();
+
+    // TODO: May still need changes to support key rotation.
```

Comment:
Is this comment still valid?

Reply:
That should be removed. I'll include this in my next commit, if any other findings.